src/test/java/uk/ac/ebi/eva/pipeline/configuration/validation/VepInputGeneratorStepParametersValidatorTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2016 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.pipeline.configuration.validation;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.batch.core.JobParameter;
+import org.springframework.batch.core.JobParameters;
+import org.springframework.batch.core.JobParametersInvalidException;
+
+import uk.ac.ebi.eva.pipeline.configuration.JobParametersNames;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * Tests that the arguments necessary to run a {@link uk.ac.ebi.eva.pipeline.jobs.steps.VepInputGeneratorStep} are
+ * correctly validated
+ */
+public class VepInputGeneratorStepParametersValidatorTest {
+    private VepInputGeneratorStepParametersValidator validator;
+
+    @Before
+    public void initialize() {
+        validator = new VepInputGeneratorStepParametersValidator();
+    }
+
+    @Test
+    public void allJobParametersAreValid() throws JobParametersInvalidException {
```

Comment:
The [JobParametersBuilder](http://docs.spring.io/spring-batch/apidocs/org/springframework/batch/core/JobParametersBuilder.html) class can be used instead of a custom Map object.